app/components/List.jsx

```diff
@@ -4,8 +4,19 @@ import React from 'react';
 import ListItem from './ListItem';
 import ReactCSSTransitionGroup from './TimeoutTransitionGroup';
 import { mergeAndPrefix } from '../utils/stylePropable';
+import { List } from 'material-ui';
+
+export default class extends React.Component {
+  static propTypes = {
```

Comment:
As much as I like the `static` keyword, it's too experimental right now. Just do as the other components do and explicitly set the statics on the constructor.

Classes will be getting replaced with stamps soon enough. :)